src/libm/sleefsimddp.c

```diff
@@ -2751,6 +2761,33 @@ EXPORT CONST void *xgetPtr(int name) {
 #include ALIAS_NO_EXT_SUFFIX
 #endif
 
+#ifdef ENABLE_GNUABI
```

Comment:
Could you please move this back to the original position? It is really hard to tell what you have changed here.